### DISCUSSION TITLE
Welcome to the Forum! (DenysAlmaral.com)

### DISCUSSION THREAD

**Discussion by piXelicidio** [piXelicidio/Hub#1] [Announcements]
## 👋 Hey there!

Welcome to this cozy corner dedicated to all things Denys Almaral! Whether you're using one of my commercial products, tinkering with the free stuff, or just interested in game development and 3D modeling, you're in the right place.

This is where we can:

- Ask for help or offer support.
- Share our experiences and ideas.
- Discuss anything related to my projects or your own creations.

We're all here to help each other create awesome games and visualizations. So, let's keep it friendly, supportive, and fun, alright? 💪

**Comment by NguyenNgocThach2001** [comment]
Hi. Your Trump models are funny and look wonderful. I spent some time studying your file, and my question is: how do you map the uv? Can you walk me through the process? Thanks!
https://sketchfab.com/3d-models/lowpoly-trump-free-character-ba7b382c918b40e6b07011da4ff3ebc0
![image](https://github.com/piXelicidio/PolyChat/assets/68904324/04cec1ef-8947-4ec9-97e2-3611f83fe4f6)

**Reply by piXelicidio** [reply]
Hi, thank you! I'm glad you like my work.

The typical technique to use textures as palette colors is explained here for Blender: https://www.youtube.com/watch?v=8NEmx0cHwoI
You select the UV faces and scale down to the square flat color.

Since I do this... A LOT, I have developed my own scripted tools for 3ds Max and Unity:
px Poly Paint for 3ds Max: https://www.youtube.com/watch?v=qEWUPV-pTmg&t=1s
DA Poly Paint for Unity: https://www.youtube.com/watch?v=wEDbnaEky0Y&t=0s

If you use 3ds Max with my script the process is very easy and fast!

**Reply by NguyenNgocThach2001** [reply]
Ah Thanks. After a day of trying, I finally know how to do it.

**Comment by dirkk0** [comment]
Hi Denys!
Quick question: what animation is included in your "Low Poly Military Squad Rigged Characters Pack" on SketchFab?
Thanks!

**Reply by piXelicidio** [reply]
Hi Dirkko,
The "Low Poly Military ... Pack" doesn't include animations. However, you can check out [this compatible free set](https://denysalmaral.com/2019/07/free-animation-pack-city-people-common-moves-wip.html) with basic idles and motions. Please note, they're not military-specific.
